### ISSUE TITLE
Asynchronous tasks should pass primary keys to task

### ISSUE THREAD

**Issue by nllong** [SEED-platform/seed#432]
Right now the background tasks are handed all the JSON data to process. The background task should only be handed the PK of the building snapshot models and request (in parallel) the data from the database to act on.

- [x] Refactor chunk code to pass ids
- [x] Remove batch method
- [x] Disable pickling of data (reenable JSON) for celery/redis

**Comment by RDmitchell** [comment]
@nllong -- would it make sense to have @rHorsey work on this ? I could assign it to him now and he could work on it as he has time in Dec and into Jan?

**Comment by RDmitchell** [comment]
@nicholasserra @nllong -- I am going to close this issue. Seems like we have discovered most of the issues associated with this change and fixed them. If we find more we can open a new issue.
